be/src/common/config.h

```diff
@@ -95,11 +95,15 @@ CONF_Int32(push_worker_count_high_priority, "3");
 
 // The count of thread to publish version per transaction
 CONF_mInt32(transaction_publish_version_worker_count, "0");
+// The min count of thread to publish version per transaction
+CONF_Int32(transaction_publish_version_worker_count_min, "0");
```

Comment:
make it dynamic mutable whenever possible.

And how this configuration differentiate to `transaction_publish_version_worker_count`?